### DISCUSSION TITLE
Upload and loading of images in the translation of the documentation in Spanish.

### DISCUSSION THREAD

**Discussion by andresbermeoq** [fastapi/fastapi#9474] [Questions] [answered]
Hi…
I was reading the documentation about the concurrency and the translation in Spanish… I don't have the explication complete and organize that exists in the official documentation.
This is the official documentation…
<img width="1587" alt="Screenshot 2023-05-01 at 19 41 40" src="https://user-images.githubusercontent.com/24640851/235555760-70ef5ab9-3529-455d-9d8e-16937f213a5b.png">
While that, in the translation, don't exist the complete explication.
<img width="1587" alt="Screenshot 2023-05-01 at 19 44 27" src="https://user-images.githubusercontent.com/24640851/235555992-e351f725-79d8-4bde-9e7e-73a86f466bd1.png">
Maybe I could try to complete the documentation.

**Comment by Kludex** [comment, marked as answer]
You can open a PR with the changes, yep.

**Reply by tiangolo** [reply]
And here are the docs for helping with translations: https://fastapi.tiangolo.com/contributing/#translations
